### DISCUSSION TITLE
recipes/holiday/aunt-marys-cherry-mashers/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#881] [Comments]
# recipes/holiday/aunt-marys-cherry-mashers/

A collection of my recipes.

https://nicholaswilde.io/recipes/holiday/aunt-marys-cherry-mashers/

**Comment by nicholaswilde** [comment]
TIP: If you can’t find or don’t want to use the powdered frosting mix, omit the mix and decrease the amount of sweetened condensed milk to 3 tablespoons. In this episode, Jo sprinkles the Cherry Mashers with multicolored nonpareils.

**Reply by nicholaswilde** [reply]
If not using frosting, try freezing instead of chilling to make firmer.
Make smaller balls, about 1 Tbsp.
Perhaps add a little bit of corn starch to thicken.
